packages/select/stories/select.stories.tsx

```diff
@@ -40,28 +52,48 @@ export const Default: Story = ({ autofill, creatable, disabled, helpMessage, isM
       autoFill1: '',
       autoFill2: '',
     }}
-    validationSchema={isMulti ? multiValueSchema('select', required, min, max) : singleValueSchema('select', required)}
+    validationSchema={
+      isMulti
+        ? multiValueSchema('select', required, min, max, !autofill)
```

Comment:
Does `autofill` determine if we should use `string` vs `object`? I feel like `raw` should tell us this instead